<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>592</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(16, 16, 255)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 152, 34);
border-radius:20</string>
   </property>
   <property name="text">
    <string>cadastro de alunos</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Script MT Bold</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20;
font: 75 15pt &quot;Script MT Bold&quot;;
border-width: 2px;
border-style: outset;
border-color: black;
background-color:rgb(255, 152, 34)</string>
   </property>
   <property name="text">
    <string>Nome do aluno:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Script MT Bold</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15;
background-color:rgb(255, 152, 34);
font: 75 15pt &quot;Script MT Bold&quot;;
border-width: 2px;
border-style: outset;
border-color: black;
</string>
   </property>
   <property name="text">
    <string>Nome do pai:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Script MT Bold</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15;
background-color:rgb(255, 152, 34);
font: 75 12pt &quot;Script MT Bold&quot;;
border-width: 2px;
border-style: outset;
background-color:rgb(255, 152, 34);
</string>
   </property>
   <property name="text">
    <string>Valor mensalidade:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Script MT Bold</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15;
background-color:rgb(255, 152, 34);
font: 75 15pt &quot;Script MT Bold&quot;;
border-width: 2px;
border-style: outset;
background-color:rgb(255, 152, 34)</string>
   </property>
   <property name="text">
    <string>turma:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15;
background-color:rgb(255, 152, 34);
font: 75 12pt &quot;Script MT Bold&quot;;
border-width: 2px;
border-style: outset;
background-color:rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15;
background-color:rgb(255, 152, 34);
font: 75 12pt &quot;Script MT Bold&quot;;
border-width: 2px;
border-style: outset;
background-color:rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>209</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15;
background-color:rgb(255, 152, 34);
font: 75 12pt &quot;Script MT Bold&quot;;
border-width: 2px;
border-style: outset;
background-color:rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>260</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15;
background-color:rgb(255, 152, 34);
font: 75 12pt &quot;Script MT Bold&quot;;
border-width: 2px;
border-style: outset;
background-color:rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>440</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Enviar</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>matutino</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>vespertino</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15;
background-color:rgb(255, 152, 34);
font: 75 15pt &quot;Script MT Bold&quot;;
border-width: 2px;
border-style: outset;
background-color:rgb(255, 152, 34)</string>
   </property>
   <property name="text">
    <string>turno:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15;
background-color:rgb(255, 152, 34);
font: 75 15pt &quot;Script MT Bold&quot;;
border-width: 2px;
border-style: outset;
background-color:rgb(255, 152, 34)</string>
   </property>
   <property name="text">
    <string>listar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
